src/main/java/address/model/ModelManager.java

```diff
@@ -46,7 +46,8 @@
      * AddressBook and its variables should not be null.
      * @param src
      */
-    public ModelManager(AddressBook src, UserPrefs prefs) {
+
```

Comment:
Should there be a new line between the method declaration and it's header comment?